python/setup.py

```diff
@@ -2,28 +2,30 @@
 from version import get_git_version
 
 setup_args = dict(
-  name = 'swiftnav',
-  version = get_git_version(),
-  description = 'Python bindings to the libswiftnav library.',
-  license = 'LGPLv3',
-  url = 'http://www.swiftnav.com',
-  author = 'Swift Navigation Inc.',
-  author_email = 'dev@swiftnav.com',
-  maintainer = 'Swift Navigation',
-  maintainer_email = 'dev@swiftnav.com',
-  packages = ['swiftnav'],
+    name='swiftnav',
```

Comment:
@ljbade @benjamin0 https://www.python.org/dev/peps/pep-0008/#id3
https://www.python.org/dev/peps/pep-0008/#fn-hi
_-Hanging indents should add a level_

The issue is related to PEP8 conformance of E121 and E126. I suggest to update the coding style to mention that hanging indents are 2 spaces. It would make it impossible to use auto-indentation, but I'm perfectly fine with that.

For example, when using pep8 to check the style with only 2 spaces in hanging indents:

    $ pep8 --ignore E111,E114 setup.py
    setup.py:5:3: E121 continuation line under-indented for hanging indent
    setup.py:55:7: E121 continuation line under-indented for hanging indent
    setup.py:63:5: E121 continuation line under-indented for hanging indent